src/lib/filesystem/src/Flow/Filesystem/Path.php

```diff
@@ -285,26 +285,26 @@ public function randomize() : self
         );
     }
 
-    public function setExtension(string $extension) : self
-    {
-        if ($this->extension) {
-            if ($this->extension === $extension) {
-                return $this;
-            }
-
-            $pathinfo = \pathinfo($this->path);
-            $path = ($pathinfo['dirname'] ?? '') . DIRECTORY_SEPARATOR . $pathinfo['filename'] . '.' . $extension;
-
-            return new self($this->protocol->scheme() . \ltrim($path, DIRECTORY_SEPARATOR), $this->options);
-        }
-
-        return new self($this->uri() . '.' . $extension, $this->options);
-    }
-
-    public function startsWith(self $path) : bool
-    {
-        return \str_starts_with($this->path, $path->path);
-    }
+    //    public function setExtension(string $extension) : self
```

Comment:
Seems this was missed.

Reply:
🤦‍♂️ thanks, will remove it in the next PR!